frontend/packages/core/src/Input/text-field.tsx

```diff
@@ -20,11 +20,13 @@ const BaseTextField = ({ InputProps, InputLabelProps, ...props }: MuiStandardTex
 );
 
 const StyledTextField = styled(BaseTextField)({
+  padding: "8px 0",
```

Comment:
another decision point here, should children have inherent padding or should the parent pad it's children (e.g. something like a `& > *` selector in containers), not sure what's best practice